api/src/database/migrations/20240515A-add-session-window.ts

```diff
@@ -0,0 +1,13 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_sessions', (table) => {
+		table.string('next_token').nullable();
```

Comment:
This should be of length `64`, as our `token` column is of length `64` as well[^1] (and I could not find any migrations touching that, but please double check me on that)

```suggestion
		table.string('next_token', 64).nullable();
```

[^1]: https://github.com/directus/directus/blob/6e7529ac0e7969562d2ab8387dd17dbbe62f6849/api/src/database/seeds/12-sessions.yaml#L4-L8

Reply:
Good catch 😄